### PR TITLE
BC-2785 tests

### PR DESCRIPTION
**Pull request by Loki-Afro**
# Description

## Links to Tickets or other pull requests
https://ticketsystem.dbildungscloud.de/browse/BC-2785
## Approval for review

- [x] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.

> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.

### CODE REVIEW

File: ansible/group_vars/all/config.yml [hunk 1]
```diff
@@ -708,11 +708,6 @@ configuration_all:
     server: false
     client: true
     nuxtclient: false
-  FEATURE_LEGACY_HYDRA_ENABLED:
```
Comment (virgilchiriac) [inline ansible/group_vars/all/config.yml:711]: this removal has to be coordinated with removal form usage in [legacy client ](https://github.com/hpi-schul-cloud/schulcloud-client/blob/main/controllers/oauth2.js#L12)
In this case, this is [set in code to true by default](https://github.com/hpi-schul-cloud/schulcloud-client/blob/main/config/default.schema.json#L552), and removing the override "false" here, would reactivate the legacy version

**Review by virgilchiriac — CHANGES_REQUESTED** [review]

File: ansible/group_vars/all/config.yml (hunk 1 above)
Comment (Loki-Afro) [reply ansible/group_vars/all/config.yml:711]: if this variable is not used anywhere it can get deleted, server, client nuxt all are set to false

**Review by SevenWaysDP — APPROVED** [review]

**Comment by virgilchiriac** [comment]
this removal of Hydra Legacy env var has to be coordinated with removal form usage in [legacy client ](https://github.com/hpi-schul-cloud/schulcloud-client/blob/main/controllers/oauth2.js#L12)
In this case, this is [set in code to true by default](https://github.com/hpi-schul-cloud/schulcloud-client/blob/main/config/default.schema.json#L552), and removing the override "false" here, would reactivate the legacy version

**Review by virgilchiriac — COMMENTED** [review]
bettermarks might be affected

**Comment by Loki-Afro** [comment]
to say it again, nothing is affect, because this variable is never actually put in any config map, therefore the defaults from the code applied and will apply

tldr;
nothing changed for the application
